python/src/wslink/backends/aiohttp/__init__.py

```diff
@@ -246,11 +246,23 @@ async def handleWsRequest(self, request):
 
         return current_ws
 
-    async def onConnect(self):
-        pass
-
-    async def onClose(self):
-        pass
+    async def onConnect(self, request, client_id):
+        if not self.serverProtocol:
+            return
+        if getattr(self.serverProtocol, "onConnect"):
```

Comment:
should we use for __if__, `hasattr` instead?

Reply:
ok.